python/sqlalchemy-openmldb/sqlalchemy_openmldb/openmldbapi/_openmldbapi.py

```diff
@@ -304,7 +304,7 @@ def execute(self, operation, parameters=()):
 
     @connected
     def executemany(self, operation, parameters=()):
-        pass
+        raise NotSupportedError("executemany() is currently unsupported in OpenMLDB")
```

Comment:
The exception message could be simpler. Just like ``raise NotSupportedError("Unsupported in OpenMLDB").

Reply:
Updated.